### ISSUE TITLE
CVPR 2016谷歌论文全收录：直击谷歌计算机视觉研究最新动态（附论文）

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/datanmachlearn#36]
# CVPR 2016谷歌论文全收录：直击谷歌计算机视觉研究最新动态（附论文）
- Author: [机器之心](http://chuansong.me/account/almosthuman2014)
- Origin: [http://chuansong.me/n/395495342802](http://chuansong.me/n/395495342802)
- Related:

1.在多人视频中检测事件以及关键参与者（Detecting events and key actors in multi-person video）

> 论文介绍的 CV 系统使用了循环神经网络来为每一帧创建一个「关注面具（attention mask）」，然后随着时间线跟踪每个目标的关联性。

2.从多个视频中发现有关节对象类别的身体部位（Discovering the physical parts of an articulated object class from multiple videos）

> 我们提出了一种基于运动的从多个视频中发现有关节对象（ articulated object ）的身体部位（如马的头/躯干和腿）的方法。其中的关键是从多个视频中，找到一个对象中相对于其它部位区域而表现出一致运动的区域。

3.明确目标描述的生成与理解（Generation and Comprehension of Unambiguous Object Descriptions）

> 我们提出了一个可以生成图片中特定目标（object）或区域明确描述（所谓的指称表达）的方法，这个方法也可以理解或解释这一指称表达，进而推断出正被描述的目标是哪一个。
> 
> 我们也提出了一个新的用于指称表达的大规模数据集，其基础是 MSCOCO。我们已经发布了这个数据集以及用于可视化和评估的工具箱，详见： https://github.com/ mjhucla/Google_Refexp_toolbox

4.DeepStereo：学习从世界图景中预测新视角（DeepStereo: Learning to Predict New Views From the World’s Imagery）

5.Blockout：分层深度网络的动态模型选择（Blockout: Dynamic Model Selection for Hierarchical Deep Networks）

6.反思计算机视觉的初始结构（Improving the Robustness of Deep Neural Networks via Stability Training）

7.通过稳定度训练提升深度神经网络的稳健性（Improving the Robustness of Deep Neural Networks via Stability Training）

8.使用 CNN 和一种特别训练的域转换实现的特定任务边缘探测的语义图像分割（Semantic Image Segmentation With Task-Specific Edge Detection Using CNNs and a Discriminatively Trained Domain Transform）
